qiskit/visualization/__init__.py

```diff
@@ -126,7 +126,7 @@
 from qiskit.visualization.transition_visualization import visualize_transition
 from qiskit.visualization.array import array_to_latex
 
-from .circuit_visualization import circuit_drawer, HAS_PIL, HAS_PDFLATEX
+from .circuit_visualization import circuit_drawer, HAS_PIL, HAS_PDFLATEX, HAS_PDFTOCAIRO
```

Comment:
Do we actually need to be exporting these? HAS_MATPLOTLIB is here mostly for backwards compatibility and not because I'd say it's something we wanted to support as part of the public interface

Reply:
I had the same comment about `HAS_PDFLATEX` originally, but then it transpires that it's useful to have these things for the tests (so they can decide whether they should skip or not).